Source/WebKit/WebProcess/WebCoreSupport/WebLocalFrameLoaderClient.cpp

```diff
@@ -2037,6 +2037,10 @@ void WebLocalFrameLoaderClient::didExceedNetworkUsageThreshold()
     if (url.isEmpty())
         return;
 
+    auto host = url.host().toStringWithoutCopying();
```

Comment:
I don't think the local variable should be a String constructed from `.toStringWithoutCopying()`. This is too risky / error prone. When something comes from `toStringWithoutCopying()`, it should be used right away, on the same line, so we understand it is safe not to copy.